x-pack/platform/plugins/shared/fleet/public/components/package_policy_delete_provider.tsx

```diff
@@ -140,6 +144,7 @@ export const PackagePolicyDeleteProvider: React.FunctionComponent<Props> = ({
           if (!!agentlessPolicy) {
             try {
               await sendDeleteAgentPolicy({ agentPolicyId: agentlessPolicy.id });
+              history.push(getPath('policies_list'));
```

Comment:
this doesn't seem to work correctly when deleting the package policy from the Integrations UI:

<img width="1752" alt="image" src="https://github.com/user-attachments/assets/33ba81f6-3d8a-49e6-bf8d-5bf50e54afb5" />
<img width="1638" alt="image" src="https://github.com/user-attachments/assets/7679981d-0abd-49ad-9615-452d8b2de4db" />

Reply:
Good catch, should be fixed